amplify/backend/api/colonycdapp/schema/schema.graphql

```diff
@@ -2020,7 +1943,7 @@ type Domain @model {
   """
   Colony ID associated with the Domain
   """
-  colonyId: ID! @index(name: "byColony", sortKeyFields: ["nativeId"])
+  colonyId: ID! @index(name: "byColony", sortKeyFields: ["nativeId"], queryField: "getDomainByColony")
```

Comment:
These indexes are fine, they won't need migrations, as they will just get created automatically when deploying, but the removals at the top, the ones you mostly moved to the new `balance` schema, are they just moved, or did you do any changes to their structure ?